tests/contrib/test_sqlalchemy.py

```diff
@@ -0,0 +1,120 @@
+# Copyright 2016 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import datetime
+import unittest
+
+from sqlalchemy import Column, create_engine, Integer
```

Comment:
I feel that having lines like

```
credentials = sqlalchemy.Column(oauth2client.contrib.sqlalchemy.CredentialsType)
```

is way more against coding style (line length). Also, I have never seen any project (SQLAlchemy or Django ORM) do this. Anyway, done.